_jobs/executive-assistant.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: job_post
+title: Executive Assistant Business Development
+open: true
+---
+
+About Compiler: Compiler is a small, woman-owned software consultancy. Compiler helps service-oriented government agencies build open source, human-centered, secure, agile solutions to support the delivery of government services that increase equity of opportunity.
+
+Our team specializes in building software applications around data for operations support and performance management including: data standards development, data engineering, data cleaning, extract transform and load (ETL) workflows, custom application programming interface (API) development and integration, data integration, data pipelines, data analytics, and data science. Our team’s subject matter expertise in transportation, housing, and homelessness provides depth and quality to the services we design and the applications we build.
+
+## Role
+
+In this role, you will ensure Compiler writes and submits high quality and competitive bids on time for opportunities that align with our core values. Bid opportunities we respond to are almost exclusively from government agencies and require detailed formal response packages. These bid opportunities are often called Request for Proposals (RFP) or Request for Qualifications (RFQ), but names and acronyms vary by government agency.
+
+This role is internal and not client-facing. You’ll report directly to founder and managing partner Vyki Englert, and collaborate with team members in marketing, operations, engineering, design, and product. You’ll have access to modern tools to help you get your work done in a remote environment. This role is located in Los Angeles as the work requires occasional access to the office.
+
+This is a full-time, salaried position (W2) with benefits. Full-time employment at Compiler is thirty (30) hours per week.
+
+
+## Responsibilities
+
+* Write proposal materials. You will be responsible for writing accurate and concise bids, often 6 to 20 pages, yourself. You will be writing about software development, data engineering, public policy, open source licensing, and public transportation. Compiler teammates will assist you in learning about these issues. You will engage team members to get the proper information, and be responsible for ultimately writing the bids in the proper format. You’ll be responsible for maintaining versions of standard narrative components, for example a description of company history or our commitment to equity. Government contracting is complicated and requires long-form narrative descriptions of our company, our prior work experiences, our qualifications or certifications, our approach to new work and more. You will also help team members keep updated resumes on file.
```

Comment:
Should the first sentence in each of these list items be bolded like in the Google doc?

> **Write proposal materials.** You will be responsible for writing accurate and concise bids, often 6 to 20 pages, yourself. You will be writing about software development, data engineering, public policy...

Reply:
I noticed we didn't use bold in any of the other job descriptions on the site so skipping it for now.

Reply:
Ok, sounds good 👍